erpnext/patches/v11_1/move_customer_lead_to_dynamic_column.py

```diff
@@ -10,5 +10,6 @@ def execute():
 	frappe.db.sql(""" UPDATE `tabQuotation` set party_name = customer WHERE quotation_to = 'Customer' """)
 
 	frappe.reload_doctype("Opportunity")
+	frappe.db.sql(""" UPDATE `tabOpportunity` set opportunity_from = enquiry_from where opportunity_from IS NULL""")
```

Comment:
You cannot rerun this patch because in new opportunities, lead column will be blank and party_name will have value. Hence it will blank the party_name column.
Also in new accounts, there will be no lead column in the database. Hence, it should check whether lead column exists.

So the correct solution will be writing a new patch with the specific use case.